README.md

```diff
@@ -52,4 +52,47 @@ A new signal that is the merged version of one signal from input 1 and one signa
   - signal D enters input 1 - notify DB
   - signal E enters input 2 - notify DE
 
-If the signals from input_1 and input_2 share an attribute, the merged signal takes the value from input_2.
+If the signals from input\_1 and input\_2 share an attribute, the merged signal takes the value from input\_2.
+
+MergeDynamicStream
+==================
+
+Take one input stream and merge signals together based on *stream*.
+
+Properties
+----------
+- expiration (timedelta) - length of time to store signal before dropping it
+- stream (str) - the expression used to determine what stream a signal is from
+- group\_by (str) - signals from the two inputs are merged by group
+
+Dependencies
+------------
+None
+
+Commands
+--------
+None
+
+Input
+-----
+Any list of signals.
+
+Output
+------
+A new signal that is the merged version of all the *streams*.
+
+- example (with no expiration)
```

Comment:
Examples should probably use 3 streams, it makes it a little more complicated but I think it's worth it. Maybe it's another example?